R/bin_time.R

```diff
@@ -211,15 +246,16 @@ bin_time <- function(occdf, bins, method = "mid", reps = 100) {
         #generate occurrence sequence for sampling
         occ_seq <- seq(from = occdf[i, "min_ma"],
                        to = occdf[i, "max_ma"],
-                       by = 0.01)
+                       by = 0.001)
+        #generate x for input probability function
+        x_prob <- seq(from = 0, to = 1, length.out = length(occ_seq))
+        # Generate probabilities
+        prob <- sapply(x_prob, fun, ...)
```

Comment:
Is there a reason we are using `sapply` here instead of just supplying `x_prob` as the first function to `fun`? I checked and it's an order of magnitude (or more) slower to use `sapply`.

Reply:
Not really! Just a blunder from me. Updated!